3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -231,7 +236,14 @@ object EnrichmentManager {
       event.page_urlquery = components.query.orNull
       event.page_urlfragment = components.fragment.orNull
     }
-
+    
+    // For Currency Conversion Enrichment
+    val trTotal = (CU.stringToDoublelike("tr_total", "tr_tt" )).toString
```

Comment:
No. `CU.stringToDoublelike` returns a `ValidatedString`. You cannot simplify stringify this. You need to work with the type.